Tests/Helper/Mocks/MockKeyboardNavigatorDelegate.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MockKeyboardNavigatorDelegate.swift
+//  KeyboardSupport-iOS
+//
+//  Created by John Davis on 12/18/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import Foundation
+import KeyboardSupport
+
+/// Mock Delegate to be used when testing the behavior of how KeyboardAutoNavigator invokes its delegate
+class MockKeyboardAutoNavigatorDelegate: KeyboardAutoNavigatorDelegate {
+    var keyboardNavigatorDidTapBackCount: Int = 0
+    var keyboardNavigatorDidTapBackLastNavigator: KeyboardAutoNavigator?
```

Comment:
Should we go ahead and make use of these "lastNavigator" properties in the tests for the `KeyboardAutoNavigator`. Should just need to add some lines like:
```swift
XCTAssertTrue(delegateMock?.keyboardNavigatorDidTapNextLastNavigator === keyboardNavigator)
```